packages/lab/src/date-picker/DatePickerPanel.tsx

```diff
@@ -35,6 +35,7 @@ export interface DatePickerPanelProps extends ComponentPropsWithoutRef<"div"> {
     selectedDate?: DateValue | { startDate?: DateValue; endDate?: DateValue }
   ) => void;
   helperText?: string;
+  visibleMonths?: 1 | 2;
```

Comment:
We need to add a test for this behaviour

Reply:
added